dev/com.ibm.ws.logging/src/com/ibm/ws/logging/internal/impl/BaseTraceService.java

```diff
@@ -464,13 +468,33 @@ public synchronized void update(LogProviderConfig config) {
         }
     }
 
-    public static void applyJsonFields(String value) {
-        if (value == null || value == "" || value.isEmpty()) {
+    public static boolean getIsServerConfigUpdate() {
+        return isServerConfigUpdate;
+    }
+
+    public static void applyJsonFields(String value, Boolean omitJsonFields) {
+
+        if (!isServerConfigSetup)
+            isServerConfigUpdate = true;
+        else
+            isServerConfigSetup = false;
+
+        if (isServerConfigUpdate) {
+            AccessLogData.resetJsonLoggingNameAliases();
+            FFDCData.resetJsonLoggingNameAliases();
+            LogTraceData.resetJsonLoggingNameAliasesMessage();
+            LogTraceData.resetJsonLoggingNameAliasesTrace();
+            AuditData.resetJsonLoggingNameAliases();
+        }
```

Comment:
this would make logging requests received during the update use the json field name defaults rather than their values from before the update happened.  Aliases need to go from their pre-update value to their post-update value without momentarily being set to their original value.